app/Http/Kernel.php

```diff
@@ -15,6 +15,10 @@ class Kernel extends HttpKernel
      */
     protected $middleware = [
         \Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode::class,
+        \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
```

Comment:
I don't think this file exists 🔍

Reply:
Oh wait it's from laravel I take it back!

Reply:
A trick! 😈